src/engine/controls/cuecontrol.cpp

```diff
@@ -883,6 +891,54 @@ void CueControl::hotcueSet(HotcueControl* pControl, double value, HotcueSetMode
         }
     }
 
+    // EveCue-Loop
+    bool TrackStem = false;
+    if (ControlObject::exists(ConfigKey(getGroup(), "stem_count"))) {
+        PollingControlProxy proxyStem(getGroup(), "stem_count");
+        TrackStem = proxyStem.get() > 1;
+    }
+
+    if (TrackStem) {
+        const QString groupBaseName = getGroup().remove("[").remove("]");
```

Comment:
you remove the [ her and add it below.

Reply:
this was my best silution to make [ChannelXStemY]  from [ChannelX]